mapsforge-core/src/main/java/org/mapsforge/core/util/Parameters.java

```diff
@@ -111,6 +107,19 @@ public enum SymbolScaling {ALL, POI}
         POLYGON_EXCEPTIONS.add(new Tag("freizeitkarte", "land"));
     }
 
+    public static final class Constants {
+
+        public static final int HILLSHADING_MAGNITUDE_DEFAULT = 128;
+        public static final int HILLSHADING_COLOR_DEFAULT = 0xff000000;
+        public static final long MAX_MEMORY_MB = Runtime.getRuntime().maxMemory() / 1000 / 1000;
+
+        /**
+         * Use AwtLuminanceShadingComposite or AlphaComposite.
+         * (2024: Deprecated, should not be used)
+         */
+        public static final boolean LUMINANCE_COMPOSITE = false;
```

Comment:
@Sublimis As it cannot be used externally, we can remove it.
And also the `AwtLuminanceShadingComposite` class.